src/operator/combineLatest.ts

```diff
@@ -59,16 +61,6 @@ export interface CombineLatestSignature<T> {
 }
 /* tslint:enable:max-line-length */
 
-/**
- * Combines the values from observables passed as arguments. This is done by subscribing
- * to each observable, in order, and collecting an array of each of the most recent values any time any of the observables
- * emits, then either taking that array and passing it as arguments to an option `project` function and emitting the return
- * value of that, or just emitting the array of recent values directly if there is no `project` function.
- * @param {...Observable} observables the observables to combine
- * @param {function} [project] an optional function to project the values from the combined recent values into a new value for emission.
- * @returns {Observable} an observable of other projected values from the most recent values from each observable, or an array of each of
- * the most recent values from each observable.
- */
 /* tslint:disable:max-line-length */
 export function combineLatestStatic<T>(v1: ObservableInput<T>, scheduler?: Scheduler): Observable<[T]>;
```

Comment:
I thought that was not reflecting latest changes (since it shows just one signatures). :)